dockstore-client/src/main/java/io/dockstore/client/cli/nested/AbstractEntryClient.java

```diff
@@ -1108,7 +1108,25 @@ private void processWesCommands(final List<String> args) {
     }
 
     /**
-     * Launches tools and workflows.
+     * Prints a warning if Docker isn't running. Docker is not always needed. If a workflow or tool uses Docker and
+     * it is not running, it fails with a cryptic error. This should make the problem more obvious.
+     */
+    void checkIfDockerRunning() {
+        ProcessBuilder pb = new ProcessBuilder("docker", "ps");
+        try {
+            Process process = pb.start(); // run a bash command
+            process.waitFor();
```

Comment:
I think you should give this a timeout. In case it takes a long time, we don't want to be blocking the launch just to display a hint. Maybe 2 seconds?

Reply:
Added 2 second timeout